packages/contracts/contracts/libs/LibStateChannelApp.sol

```diff
@@ -9,8 +9,8 @@ contract LibStateChannelApp {
   // The status of a challenge in the ChallengeRegistry
   enum ChallengeStatus {
     NO_CHALLENGE,
-    CHALLENGE_IS_OPEN,
-    CHALLENGE_WAS_FINALIZED
+    FINALIZES_AFTER_DEADLINE,
```

Comment:
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll what do you think of `WILL_FINALIZE_AFTER_DEADLINE` and `HAS_FINALIZED` respectively?

Reply:
I think `WILL_FINALIZE_AFTER_DEADLINE` is slightly inaccurate when the deadline has already passed